src/coremods/core_user/umode_s.cpp

```diff
@@ -146,7 +146,7 @@ std::string ModeUserServerNoticeMask::ProcessNoticeMasks(User* user, const std::
 	// Form: "+ABc-de"
 	std::string output;
 	if (plus.length() > 1)
-		output = plus;
+		output = std::move(plus);
```

Comment:
The v3 branch has to build as GNU++03 so `std::move` can't be used. This can be committed to master though.

Reply:
@SadieCat, fix it.
Commit info here:
https://github.com/inspircd/inspircd/pull/1879/commits/b4e0c5b6212def02386af0a555445412eed6f1f4